### PR TITLE
Expose information about scheduling latency in scalability tests.

### PR DESCRIPTION
**Pull request by wojtek-t**
@gmarek @davidopp

### CODE REVIEW

**Comment by wojtek-t** [comment]
@davidopp - this PR is regarding our discussion yesterday; I will fix the remaining TODOs in a separate PR, but this is just to have something working.

File: test/e2e/metrics_util.go [hunk 1]
```diff
@@ -233,6 +244,56 @@ func getMetrics(c *client.Client) (string, error) {
...
 }
 
+// Retrieves scheduler metrics information.
+func getSchedulingLatency() (SchedulingLatency, error) {
+	result := SchedulingLatency{}
+
+	cmd := "curl http://localhost:10251/metrics"
+	stdout, _, code, err := SSH(cmd, getMasterHost()+":22", testContext.Provider)
+	if err != nil || code != 0 {
+		return result, fmt.Errorf("unexpected error (code: %d): %#v", code, err)
```
Comment (gmarek) [inline test/e2e/metrics_util.go]: add 'in ssh connection with master' or sth similar.

**Comment by gmarek** [comment]
LGTM.

File: test/e2e/metrics_util.go (hunk 1 above)
Comment (wojtek-t) [inline test/e2e/metrics_util.go]: Done.

**Comment by davidopp** [comment]
Thanks!

**Comment by gmarek** [comment]
@k8s-bot test this

**Pull request merged**